orca-clouddriver/orca-clouddriver.gradle

```diff
@@ -46,6 +46,7 @@ dependencies {
   testImplementation(project(":orca-test"))
   testImplementation(project(":orca-test-groovy"))
   testImplementation(project(":orca-api-tck"))
+  testImplementation(project(":orca-queue"))
```

Comment:
This PR for me without this addition.

Reply:
It doesn't for me - are you running the tests? The `ExpressionAware` interface that I'm using in the tests is in `:orca-queue`.

Reply:
I ran `$ ./gradlew build` , but maybe I something was cached.  I ran it again after clean, and yes, this is necessary.  Sorry for the trouble.